pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -140,12 +140,12 @@ var _ = Describe("Generating fluentd config", func() {
 				@type tail
 				@id container-input
 				path "/var/log/containers/*.log"
+				exclude_path ['fluentd-*_openshift-logging_*.log', 'elasticsearch-*_openshift-logging_*.log']
```

Comment:
we should also exclude kibana, currently it writes to stdout and it can be noisy

Reply:
done